contrib/sarplus/python/setup.py

```diff
@@ -28,21 +43,24 @@ def __str__(self):
         "Programming Language :: Python :: 3.4",

```

Comment:
In the rest of the repo we have just upgraded to Dev status 5, see https://github.com/microsoft/recommenders/pull/1579. @simonzhaoms do you think we should upgrade as well in SAR+?

Reply:
In `recommenders` the oldest version of Python we support is 3.6. Can `pysarplus` support 3.4 and 3.5 without a problem?

Reply:
I didn't notice this.  The oldest version being tested in the workflow is 3.6: https://github.com/simonzhaoms/recommenders/blob/34c72f73934474c7d4f8d4cb7be0e054350e1f55/.github/workflows/sarplus.yml#L32

Reply:
Remove python 3.4 and 3.5, and upgraded to Dev status 5